tests/integration/test_application.py

```diff
@@ -503,6 +503,9 @@ def test_runtime_error_logging(monkeypatch, tmp_path, create_app, mocker):
         "craft_application.services.lifecycle._get_parts_action_message",
         side_effect=runtime_error,
     )
+    # Override the default of setting debug - here we're explicitly that we return
```

Comment:
I am not sure why but I cannot submit add suggestions on this PR (maybe I do not have write access to the project).

`here we're explicitly **checking** that we return`

Reply:
Good catch, thanks! I've actually deleted this line now in favour of having a `production_mode` fixture :-)